pkg/controller/worker/machines.go

```diff
@@ -207,7 +207,7 @@ func (w *workerDelegate) allocateDataVolumes(ctx context.Context) error {
 	}
 
 	for _, machineClass := range w.machineClasses {
-		if !machineClass["dontUsePreAllocatedDataVolumes"].(bool) {
+		if !machineClass["useDynamicallyAllocatedDataVolumes"].(bool) {
```

Comment:
in this case you should remove negation in this condition, should be 
```
if machineClass["useDynamicallyAllocatedDataVolumes"].(bool) {
```

Reply:
No, I shouldn't. The semantic of the option is the same, only the name has changed.

Reply:
@stoyanr but this means(from a semantic point of view) that the default is actually using the dynamic data volume allocation(dv created on vm creation).

Reply:
so in that case if somebody will set `useDynamicallyAllocatedDataVolumes = true` they won't use it as there is negation.